app/index.js

```diff
@@ -53,23 +53,26 @@ module.exports = yeoman.Base.extend({
           checked: true
         }, {
           name: 'Framework (Bootstrap 4)',
-          value: 'bootstrapSass',
+          value: 'bootstrap',
           checked: true
         }, {
           name: 'Tests (Mocha, Casperjs and Chai)',
           value: 'tests',
           checked: false
+        }, {
+          name: 'Contentful',
```

Comment:
In this case, what is the purpose of using ContentFull if you are not using the stylguide ? I think we have to keep it inside the condition.

Ex : 
❎ Stylguide
✅ ContentFull